Source/WebKit/Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.h

```diff
@@ -25,11 +25,14 @@
 
 #pragma once
 
+#import <objc/objc.h>
+#import <xpc/xpc.h>
```

Comment:
I wonder if we could do these imports in the implementation files, since the new declaration just needs a definition for `xpc_conneciton_t`, which we could forward declare. I think these are fairly big headers, so avoiding them in this header could be a nice build-time benefit.

Reply:
That is a good point! I tried doing the forward declaration, although it became a bit complicated since the definition of `xpc_connection_t` depends on the language. If you are ok with it, I'll keep the include for now.